exosphere-shared/src/docker-helper.ls

```diff
@@ -48,6 +48,38 @@ class DockerHelper
       done null, map((.RepoTags |> head |> (.split ':') |> head), images)
 
 
+  @get-dangling-images = (done) ->
+    docker.list-images {"filters": '{"dangling": ["true"]}'}, (err, images) ->
```

Comment:
Does this work: `docker.list-images {filters: {dangling: [true]}}, (err, images) ->` ?